docs/setting_up_your_development_environment.md

```diff
@@ -1,42 +1,40 @@
 Setting up Your Hypothesis Development Environment
 ==================================================
 
-Set up your GitHub account and SSH key
---------------------------------------
+This guide will walk you through setting up your Hypothesis development environment: installing everything and getting the apps running locally.
 
-You need to be able to `git clone` private repositories from the
-[hypothesis organization on GitHub](https://github.com/hypothesis/):
+You will need
+-------------
 
-1. [Sign up for a free GitHub account](https://github.com/signup) if you don't already have one
-2. Get someone to add your GitHub account to the `hypothesis` organization if it hasn't already been added
-3. [Set up an SSH key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
-   for git to use to authenticate to GitHub
+1. A GitHub.com account that is a member of the `hypothesis` organization. If you've been hired by Hypothesis this should already have been set up for you.
 
 Install prerequisites
 ---------------------
 
-You'll need:
+You need to install:
 
 1. [Git](https://git-scm.com/)
-2. [Node.js](https://nodejs.org/en/)
-3. [Yarn 1 (Classic)](https://classic.yarnpkg.com/)
-4. [Docker](https://www.docker.com/)
-5. [pyenv](https://github.com/pyenv/pyenv)
-6. [GNU Make](https://www.gnu.org/software/make/)
-7. [pg_config](https://www.postgresql.org/docs/current/app-pgconfig.html)
+2. [GitHub CLI](https://cli.github.com/)
+3. [Node.js](https://nodejs.org/en/)
+4. [Yarn 1 (Classic)](https://classic.yarnpkg.com/)
+5. [Docker](https://www.docker.com/)
+6. [pyenv](https://github.com/pyenv/pyenv)
+7. [GNU Make](https://www.gnu.org/software/make/)
+8. [pg_config](https://www.postgresql.org/docs/current/app-pgconfig.html)
 
 <details>
 <summary>Installing the prerequisites on macOS</summary>
 
 1. Install [Homebrew](https://brew.sh/)
 2. Run:
    ```terminal
-   brew install git node postgresql pyenv
+   brew install git gh node postgresql pyenv
    npm install --global yarn
    ```
 3. Follow [Docker's install instructions](https://docs.docker.com/get-docker/). You **don't** need to install Docker Compose
-4. Follow pyenv's instructions to [set up your shell for pyenv](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv)
-5. Follow pyenv's instructions to [install Python build dependencies](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)
+4. Install pyenv's shell integration and build dependencies. The `brew` command above will have installed pyenv itself but you still need to:
+   1. Follow pyenv's instructions to [set up your shell for pyenv](https://github.com/pyenv/pyenv#set-up-your-shell-environment-for-pyenv)
+   2. Follow pyenv's instructions to [install Python build dependencies](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)
```

Comment:
These docs were a bit confusing before because it was easy to miss that `pyenv` was already installed by the `brew install` command above but you still just need to install the shell integration and build dependencies. Re-worded it to clarify that.